src/utils/aLinkCreator/src/md4.cpp

```diff
@@ -327,7 +327,7 @@ wxString MD4::calcMd4FromFile(const wxString &filename, MD4Hook hook)
       char *buf = new char[bufSize];
 
       bool keep_going = true;
-      size_t read = 0;
+      size_t read;
```

Comment:
You can also reduce the scope of `read` to the `if` block using it.

Reply:
ok, done